gridstatus/ercot.py

```diff
@@ -1773,6 +1777,49 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        # just so we can share parse logic
+        kwargs = dict(
+            report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+            date=date,
+            verbose=verbose,
+            extension="csv",
+        )
+
+        if date == "latest":
+            doc = self._get_document(**kwargs)
+            docs = [doc]
+        else:
+            docs = self._get_documents(**kwargs)
+
+        df = pd.concat([pd.read_csv(i.url, compression="zip") for i in docs])
+
+        df["SCEDTimeStamp"] = pd.to_datetime(df["SCEDTimeStamp"]).dt.tz_localize(
+            self.default_timezone,
+            ambiguous=df["RepeatedHourFlag"] == "Y",
+        )
+
+        df["System Lambda"] = df["SystemLambda"].astype("float64")
+
+        df.drop("SystemLambda", axis=1, inplace=True)
```

Comment:
the final column names should be just: "SCED Time Stamp", "System Lambda". I dont think that is what this code does or what you are testing for